controllers/profiles/reconciler_dev.go

```diff
@@ -324,11 +358,108 @@ func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVis
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = make([]v1.VolumeMount, 0)
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = volumeMounts
 
+			for keycm, cm := range externalResourceConfigMaps {
+				deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes,
```

Comment:
Please use `volumes = append` instead. Two lines above you have the override logic.

Reply:
ok

Reply:
ok

Reply:
ok